tests/timeseries/test_quantities.py

```diff
@@ -45,6 +48,48 @@ class Fz2(
     """Frequency quantity with broad exponent unit map."""
 
 
+_CtorType = Callable[[float], Quantity]
+
+# This is the current number of subclasses. This probably will get outdated, but it will
+# provide at least some safety against something going really wrong and end up testing
+# an empty list. With this we should at least make sure we are not testing less classes
+# than before. We don't get the actual number using len(_QUANTITY_SUBCLASSES) because it
+# would defeat the purpose of the test.
+_SANITFY_NUM_CLASSES = 7
```

Comment:
A bit late to the review, but it might be better to call this `_SANITIFY_NUM_CLASSES`.. I just don't see a reason to skip this extra I...

Reply:
just try to pronounce it with and without :P

Reply:
I have no idea how I ended up typing this. But luckily we have smart completion, so typos are properly propagated and the code never fails!